invenio_stats/aggregations.py

```diff
@@ -360,18 +385,49 @@ def agg_iter(self, dt):
                         aggregation_data[destination] = doc[source]
                     else:
                         aggregation_data[destination] = source(doc, aggregation_data)
-
-                index_name = "stats-{0}-{1}".format(
-                    self.event, interval_date.strftime(self.index_name_suffix)
-                )
-
                 yield {
                     "_id": "{0}-{1}".format(
                         aggregation["key"], interval_date.strftime(self.doc_id_suffix)
                     ),
-                    "_index": prefix_index(index_name),
+                    "_index": index_name,
                     "_source": aggregation_data,
                 }
+                yield {
+                    '_index': prefix_index("buffer_stats_to_reindex"),
+                    '_id': doc["parent_recid"],
+                    '_source': {
+                        "timestamp":  datetime.utcnow(),
+                    },
+                }
+        #THIS CALL (AND THE FOLLOWING METHOD) SHOULD BE MOVED SOMEWHERE ELSE
```

Comment:
This should be moved to other module... I'm leaving it here for the time being to see if you like the implementation (which requires the `buffer` index)